test-assets/test-projects/NUnitTestProject/NuGet.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="AppVeyor NUnit CI Feed" value="https://ci.appveyor.com/nuget/nunit" />
```

Comment:
Minor, but you don't need this line/feed unless you want to pull in pre-release versions of the NUnit Framework. The adapter PR only uses the Engine and Adapter feeds.